src/text/text3.asm

```diff
@@ -639,7 +639,7 @@ Text02f8: ; 3d3e0 (f:53e0)
 	line "Attack              To previous page"
 	done
 
-Text02f9: ; 3d48f (f:548f)
+ChooseWordInGlossaryText: ; 3d48f (f:548f)
```

Comment:
In general, I like when the text label mimics the actual text as much as possible. If you added the word "Glossary" to the label because this belong to some group of labels related to each other and to the Glossary feature, then it's fine to use the "GlossaryText" in all of them. But still, I would rename it to ``ChooseWordAndPressAButtonText``, or to ``ChooseWordPressAButtonGlossaryText`` if the aforementoned applies.

Reply:
Okay. I thought it might be a little bit odd if I lengthen it too long, but as it's acceptable, I will follow your suggestion.